app/controllers/journals_controller.rb

```diff
@@ -101,9 +101,8 @@ def field_param
     @field_param ||= params[:field].parameterize.underscore
   end
 
-  # Is this a valid field for diff'ing?
-  def valid_field?
-    field_param == 'description'
+  def valid_field_for_diffing?
```

Comment:
I appreciate that you removed the comment and updated the method name to make the intention clearer :+1: